### PR TITLE
MKS - Backing up with Velero - Added information of install newer versions from 1.14

### PR DESCRIPTION
**Pull request by mas-synaigy**

### CODE REVIEW

File: pages/public_cloud/containers_orchestration/managed_kubernetes/backing-up-cluster-with-velero/guide.en-ie.md [hunk 1]
```diff
@@ -141,6 +141,24 @@ velero install \
   --snapshot-location-config region=<public cloud region without digit>,enableSharedConfig=true
 ```
 
+> [!primary]
+>
+> Starting with version 1.14 the plugin-for-csi is integrated in velero. We can simply remove it from the install example if we install a version 1.14 or newer. For upgrading an older version follow the upgrade notes [Upgrade-to-1.14](https://velero.io/docs/v1.14/upgrade-to-1.14/).
```
Comment (Y0Coss) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/backing-up-cluster-with-velero/guide.en-ie.md]:
```suggestion
> Starting with version 1.14 the plugin-for-csi is integrated in velero. You can simply remove it from the install example if you install version 1.14 or newer. For upgrading an older version follow the upgrade notes: [Upgrade-to-1.14](https://velero.io/docs/v1.14/upgrade-to-1.14/).
```

**Review by antonin-a — APPROVED** [review]

**Comment by Y0Coss** [comment]
Thank you for your contribution @mas-synaigy 
Duplication of this update for other alternates of this guide will be done through a 2nd Pull Request.

**Comment by mas-synaigy** [comment]
Thank you @Y0Coss! Do I need to make the changes or is this 2nd step, update of other guides, done automatically?

**Comment by Y0Coss** [comment]
I'll handle this on my side very soon.

**Pull request merged**